lib/miq_unicode.rb

```diff
@@ -0,0 +1,43 @@
+module ManageIQ
```

Comment:
Can this be a little more specific like `ManageIQ::UnicodeString`?  The `using ManageIQ` all over feels a bit scary as that's a top-level module

Reply:
Ok, scoped it to `ManageIQ::UnicodeString` as per your suggestion.